How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+This doc contains the internal Basho policy for building, testing, and releasing Riak.
+
+# Scheduling
+All public releases must conform to the following schedule:
+* An expected delivery date on a Wednesday. Each delivery date must have at least one fallback date that is no earlier than one week after the expected delivery date.
+* A Go/No-Go scheduled on the Thursday after the expected delivery date or fallback date to ensure that the public release can proceed as planned. During the Go/No-Go we must ensure:
+  * Giddyup has been triaged, and there are no major issues
+  * Smoke testing has completed successfully on all supported platforms
+  * Documentation content has been merged and triaged if necessary, and release notes are ready to ship
+  * Packages for all supported platforms have been built and are ready to stage
```

Comment:
The review that I think is most helpful in that regard is when they are actually staged in the download locations, which I have as a different step. I want the go/no-go to happen before staging so I don't have to double up efforts if we back out at this point. I can explicitly call that out in the staging step.